zero_bin/rpc/src/native/txn.rs

```diff
@@ -209,6 +211,28 @@ fn process_nonce(
         })
 }
 
+/// Processes the self destruct for the given account state.
+fn process_self_destruct(
```

Comment:
nit: returning an Option<bool> is a bit cryptic - could you document what it means?

Reply:
Also you never return `Some(false)`, which feels odd